### PR TITLE
Feature/pdct 988 add autotag release to nav frontend

### PR DESCRIPTION
**Pull request by katybaulch**
# What's changed

Add auto tag and release to navigator frontend. 

## Proposed version

Please select the option below that is most relevant from the list below. This
will be used to generate the next tag version name during auto-tagging.

- [x] Skip auto-tagging
- [ ] Patch
- [ ] Minor version
- [ ] Major version

Visit the [Semver website](https://semver.org/#summary) to understand the
difference between `MAJOR`, `MINOR`, and `PATCH` versions.

Notes:

- If none of these options are selected, auto-tagging will fail
- Where multiple options are selected, the most senior option ticked will be
  used -- e.g. Major > Minor > Patch
- If you are selecting the version in the list above using the textbox, make
  sure your selected option is marked `[x]` with no spaces in between the
  brackets and the `x`

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -15,6 +15,7 @@ app.prepare().then(() => {
     handle(req, res, parsedUrl);
   }).listen(3000, (err) => {
     if (err) throw err;
+    /* trunk-ignore(eslint/no-console) */
```
Comment (THOR300) [inline server.js:18]: Why is this line needed!?

**Review by THOR300 — APPROVED** [review]
LGTM

File: server.js (hunk 1 above)
Comment (katybaulch) [reply server.js:18]: Because ESLint has a no-console rule to avoid people leaving in debug statements, and the line below has a debug statement that we want to leave in (have checked this with Patrick).
